### PR TITLE
Allow symfony/contract 3.* & psr/container 2.*

### PR DESCRIPTION
**Pull request by benji07**
In order to update my project in Symfony 6.2 i need `symfony/contract` 3.x & `psr/container 2.x`.

I also remove `sensiolabs/security-checker` as it is not used anymore.

Some change have been updated after running your php-cs-fixer configuration

### CODE REVIEW

**Review by Oliboy50 — APPROVED** [review]
if it works without changing anything else => 🆗

**Comment by benji07** [comment]
Hey, any news about merging this PR ?

**Comment by Oliboy50** [comment]
oops... I just realized that we were waiting for @BedrockStreaming/monitoring-qa to approve before merging this PR... 

anyway... it has been released in [v8.0.2](https://packagist.org/packages/m6web/statsd-prometheus-bundle#v8.0.2), sorry you had to wait

**Pull request merged**
